qa/automation/CareerPortal.Tests.Scenarios/2. Extended example/homepage/name_field_validation/email_field_validation/Send_cv/AtachCv.feature

```diff
@@ -0,0 +1,131 @@
+Feature: 1.1.3 Attach CV and Send 
```

Comment:
Common mitakes;
1. Incorrect separating functionality for Feature files.
All your scenarios cover form "Send CV" form and you should have only one feature file. 
2. No formatting 
3. No attached test files